### DISCUSSION TITLE
Welcome to Family-Trees-List Discussions!

### DISCUSSION THREAD

**Discussion by iyoua** [iyoua/Family-Trees-List#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hartenthaler** [comment]
I like your module. Do you know that you can add internal variables to an html-block on the home page of a tree?

For example I'm using:

```
webtrees #webtreesVersion#
#totalUsers# Benutzer angelegt

Stammbaum #gedcomFilename# zuletzt geändert am #gedcomUpdated#
#hitCount# Seitenbesuche
#totalIndividuals# Personen (davon #totalLiving# lebend; #totalSexFemales# Frauen und #totalSexMales# Männer)
#totalFamilies# Familien
#totalEvents# Einzelereignisse
#totalMedia# Medienobjekte (#totalMediaPhoto# Fotos; #totalMediaDocument# Dokumente; #totalMediaTombstone# Grabsteine; #totalMediaCertificate# Urkunden)
#totalSources# Quellen
#totalPlaces# Orte (nicht Ortsdatensätze)
```

**Reply by iyoua** [reply]
Yes,I know.

**Comment by iyoua** [comment]
These statistical tags provide rich statistical information for custom genealogy homepage and also reflect flexibility.
But they are statistics for the current tree.
WEBTREES serves the special field of genealogy, and its functions are quite perfect, with excellent professionalism in content and data security. Due to these characteristics, the audience is very small, and many people visit it once and never come back. I think it is also important to enhance website stickiness while ensuring privacy and data security.

**Comment by iyoua** [comment]
Several effects of version 1.3.3:
![capsule](https://github.com/iyoua/Family-Trees-List/assets/102014216/8fce766b-5641-4c15-95b2-6f0ba1f9d937)
![cards](https://github.com/iyoua/Family-Trees-List/assets/102014216/ff239166-3b00-47f6-b609-8242953b7b98)
![list](https://github.com/iyoua/Family-Trees-List/assets/102014216/4c94f5cd-2499-43e9-bcf7-4986d9a691b3)
![navbar](https://github.com/iyoua/Family-Trees-List/assets/102014216/898d7eec-e9e4-410d-8d33-406227eed23d)
![table](https://github.com/iyoua/Family-Trees-List/assets/102014216/666f99c7-7851-4500-8539-94772042e8ac)
